### ISSUE TITLE
[renamerOnUpdate] Bug: field_replacer being applied to entire filename when replacing a period

### ISSUE THREAD

**Issue by echo6ix** [stashapp/CommunityScripts#96]
`field_replacer` is being applied to the full filename instead of just the specified field(s).

For example, the following is stripping out all the periods from the entire filename instead of just their respective fields.
```
field_replacer = {
    "$studio": {"replace": ".", "with": ""},
    "$title": {"replace": ".", "with": ""},
    "$performer": {"replace": ".", "with": ""}
}
```

**Comment by echo6ix** [comment]
This only seems to be happening when the string being replaced is a `.`. I cannot reproduce this bug with any other string/character, though my testing hasn't been extensive.

**Comment by bnkai** [comment]
Fixed by #107
Closing
